lib/htsget-server/docker-compose.yml

```diff
@@ -44,6 +44,7 @@ services:
       HTSGET_TEST_KEY: "hoodlebug"
       HTSGET_URL: ${TYK_LOGIN_TARGET_URL}/${TYK_HTSGET_API_LISTEN_PATH}
       OPA_URL: ${OPA_PRIVATE_URL}
+      OPA_SITE_ADMIN_KEY: ${OPA_SITE_ADMIN_KEY} # only required if not equal to "site_admin"
```

Comment:
If this is a secret, it should be provided via `secret` and not `environment`

Reply:
It's not a secret, it's just the value "site_admin" usually, but I wanted to leave it open for a site to have a different role name if they choose.